docs/kubectl_update.md

```diff
@@ -28,6 +28,7 @@ $ cat pod.json | kubectl update -f -
 ```
   -f, --filename=[]: Filename, directory, or URL to file to use to update the resource.
   -h, --help=false: help for update
+      --patch="": A JSON document to override the existing resource. The resource is downloaded, patched with the JSON, then updated.
 ```
```

Comment:
Can you re-add a `--patch` example? Either marking a node unschedulable or updating the image are reasonable ones.